bitmap-steven-hawa-caleb/lib/bluescale-colors.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const fs = require('fs');
+const bmp = require(`${__dirname}/../model/bitmap-constructor.js`);
+
+module.exports = exports = {};
+
+exports.blueScaleColorTable = function(callback) {
+  fs.readFile(`${__dirname}/../../img/palette-bitmap.bmp`, function(err, data) {
+    var blueScaleBuffer = Buffer.from(data);
+    bmp(function(err, data) {
+      for (var index = 0; index < data.colorTable.length; index += 4) {
+        var blueAvg = parseInt(blueScaleBuffer[54 + index] + blueScaleBuffer[55 + index] + blueScaleBuffer[56 + index] / 3);
+        if (blueAvg > 255) blueAvg = 255;
+        blueScaleBuffer[54 + index] = blueAvg;
+      }
+      callback(null, blueScaleBuffer);
```

Comment:
@maschigokae @BatemanVO @abdih17 great use of the NodeJS `(err, result)` callback pattern along with passing in a `null` value as the error parameter